lib/matplotlib/layout_engine.py

```diff
@@ -205,6 +205,8 @@ def __init__(self, *, h_pad=None, w_pad=None,
                  hspace=mpl.rcParams['figure.constrained_layout.hspace'])
         # set anything that was passed in (None will be ignored):
         self.set(w_pad=w_pad, h_pad=h_pad, wspace=wspace, hspace=hspace)
+        # see CompressedLayoutEngine below...
```

Comment:
This comment seems wrong now?

Reply:
Ooops, yes, I decided against needing a subclass here.